torch/nested/_internal/ops.py

```diff
@@ -862,3 +862,28 @@ def embedding_default(func, *args, **kwargs):
     return NestedTensor(
         func(weight, indices._values, **new_kwargs), **extract_kwargs(indices)
     )
+
+
+@register_jagged_func(torch.ops.aten.values.default, "self: jt")
+def values_default(func, *args, **kwargs):
+    _, new_kwargs = normalize_function(
+        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+    )
+
+    inp = new_kwargs.pop("input")
+
+    return inp._values.detach()
```

Comment:
We'd eventually need to worry about some variation of https://github.com/pytorch/pytorch/issues/112024#issuecomment-1779554292, but adding a note would be okay for now.